Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -4,7 +4,6 @@
     children:
     - !type:NestedSelector # DeltaV
       tableId: LockerFillQuarterMasterDeltaV
```

Comment:
where did the spare logi channel keys go

Reply:
Damn I really did delete the wrong one